app/models/validations/household_validations.rb

```diff
@@ -36,7 +36,7 @@ def validate_armed_forces(record)
   end
 
   def validate_pregnancy(record)
-    if (record.has_pregnancy? || record.pregnancy_refused?) && !women_of_child_bearing_age_in_household(record)
+    if (record.has_pregnancy? || record.pregnancy_refused?) && women_in_household(record) && !women_of_child_bearing_age_in_household(record)
```

Comment:
Why do we need to add this condition? Isn't it already covered by the next method?

Reply:
Because if there are no women in the household but we don't want to throw an error. If there are no women `!women_of_child_bearing_age_in_household` will return true and the error will get thrown